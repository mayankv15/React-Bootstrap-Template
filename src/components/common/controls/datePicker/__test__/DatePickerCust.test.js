import React from "react";
import { render } from "@testing-library/react";
import DatePickerCust from "../index"

describe("Renders DatePickerCust", () => {
    it("should render DatePickerCust correctly", () => {
        const component = render(<DatePickerCust />);
        expect(component).toMatchSnapshot();
    });
});
