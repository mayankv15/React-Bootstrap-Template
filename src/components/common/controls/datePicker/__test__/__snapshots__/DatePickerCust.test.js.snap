// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders DatePickerCust should render DatePickerCust correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="form-control undefined"
        id="custom-date-picker"
      >
        <div
          class="react-datepicker-wrapper"
        >
          <div
            class="react-datepicker__input-container"
          >
            <div
              class="form-control"
            >
              <div
                class="MuiFormControl-root MuiTextField-root"
              >
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart"
                >
                  <div
                    class="MuiInputAdornment-root MuiInputAdornment-positionStart"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 3h-1V1h-2v2H7V1H5v2H4c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 18H4V8h16v13z"
                      />
                    </svg>
                  </div>
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input MuiOutlinedInput-input makeStyles-inputStyle-2 MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart"
                    placeholder="20/11/2021"
                    type="text"
                    value=""
                  />
                  <fieldset
                    aria-hidden="true"
                    class="PrivateNotchedOutline-root-6 MuiOutlinedInput-notchedOutline"
                    style="padding-left: 8px;"
                  >
                    <legend
                      class="PrivateNotchedOutline-legend-7"
                      style="width: 0.01px;"
                    >
                      <span>
                        ​
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="form-control undefined"
      id="custom-date-picker"
    >
      <div
        class="react-datepicker-wrapper"
      >
        <div
          class="react-datepicker__input-container"
        >
          <div
            class="form-control"
          >
            <div
              class="MuiFormControl-root MuiTextField-root"
            >
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart"
              >
                <div
                  class="MuiInputAdornment-root MuiInputAdornment-positionStart"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M20 3h-1V1h-2v2H7V1H5v2H4c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 18H4V8h16v13z"
                    />
                  </svg>
                </div>
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiOutlinedInput-input makeStyles-inputStyle-2 MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart"
                  placeholder="20/11/2021"
                  type="text"
                  value=""
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-6 MuiOutlinedInput-notchedOutline"
                  style="padding-left: 8px;"
                >
                  <legend
                    class="PrivateNotchedOutline-legend-7"
                    style="width: 0.01px;"
                  >
                    <span>
                      ​
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
